classes/Notices/Notices.php

```diff
@@ -243,39 +242,6 @@ public function deactivate_plugin() {
 		wp_send_json_success();
 	}
 
-	/**
-	 * Renew the "almost-over-quota" notice when the consumed quota percent decreases back below 80%.
-	 *
-	 * @since 1.7
-	 */
-	public function renew_almost_over_quota_notice() {
```

Comment:
this method is public, do we need to deprecate it? I don't think so, just wondering to discuss.

Reply:
I don't think that's necessary